src/ui/js/Overview.js

```diff
@@ -94,6 +94,14 @@ Overview.getOverview = function(callback) {
 Overview.showPage = function() {
   Overview.page = $('<div>');
 
+  var gameCountText;
+  var numGames=Api.new_games.nGames;
+  numGames+=Api.active_games.games.awaitingPlayer.length;
+  if (numGames > 0) {
+    gameCountText='('+numGames+') ';
+  }
+  $('title').html(gameCountText+'Button Men Online');
```

Comment:
Ditto: this entire block has no horizontal whitespace in variable definitions, or on other side of `+` in string assembly, and i'd like to see it in all those places to match the existing code.